connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -121,4 +124,23 @@ public void reconfigure(Map<String, String> props) {
      * Stop this connector.
      */
     public abstract void stop();
+
+
+    /**
+     *
+     * @return
+     */
+    public ConfigDef configDef() {
```

Comment:
Can we get rid of this? Seems like we shouldn't need it -- isn't `validate` enough?

If we do need it, we should fill out the javadoc.